README.md

```diff
@@ -29,7 +29,7 @@ All configurations in `apisix-ingress-controller` are defined with Kubernetes CR
 
 The technical architecture of `apisix-ingress-controller`:
 
-<img src="./docs/assets/images/module-0.png" alt="Architecture" width="600" height="313" />
+<img src="./docs/assets/images/module_0.png" alt="Architecture" width="600" height="313" />
```

Comment:
Excuse me, I found the `./docs/assets/images/module_0.png` in your branch is the [pic in design](https://github.com/apache/apisix-ingress-controller/blob/master/docs/assets/images/modules.png), but why ?

Reply:
What else i was supposed to do? Can you please elaborate?

Reply:
Do i have to remove the previous architecture diagram and replace with new one using “module-0” name?

Reply:
Here is the ingress-controller architecture diagram, which introduces the role of ingress-controller and its deployment position in Kubernetes, not the division of modules.

I suggest refer to this [issue](https://github.com/apache/apisix-ingress-controller/issues/274) to modify the image.

Thanks.

Reply:
Actually now the internal of apisix-ingress-controller has some changes, the diagram cannot reflect the real situation.

Reply:
where can i understand and know the latest architecture? @tokers

Reply:
We are writing them and will push them to GitHub as soon as possible :)
Also, Would you like to use this [image](https://user-images.githubusercontent.com/6814606/109374178-521b4300-78ee-11eb-8315-4da896383f53.png) to replace the current one? It's more clear (though it's a high level interpretation).